Kentico/devnet.kentico.com
# DEVREL-193 Set up a new submission process for Kentico EMS extensions

### Motivation

Allow people to submit a Kentico EMS extension to the new marketplace,

### How to test

Follow the readme and try to run the check and check if something is not missing in README.
